lib/lazy_high_charts/layout_helper.rb

```diff
@@ -106,11 +106,12 @@ def encapsulate_js(core_js)
         #{js_end}
         EOJS
       # Turbolinks >= 5
-      elsif defined?(Turbolinks) && request_turbolinks_5_tureferrer?
+      elsif defined?(Turbolinks) && is_turbolinks_5?
         js_output =<<-EOJS
         #{js_start}
-          document.addEventListener("turbolinks:load", function() {
+          document.addEventListener("turbolinks:load", function(e) {
             #{core_js}
+          	e.target.removeEventListener(e.type, arguments.callee);
```

Comment:
The arguments.callee property contains the currently executing function. [doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments/callee)

you can always use:
`var f = function(){
            document.removeEventListener('turbolinks:load', f, true);
            #{core_js}
          };
          document.addEventListener('turbolinks:load', f, true);`
does the same thing.